rpa-worker-application/src/main/resources/META-INF/application.sample.properties

```diff
@@ -30,6 +30,14 @@
 #camunda.client.zeebe.base-url=https://zeebe.camunda
 #camunda.rpa.zeebe.auth-endpoint=https://camunda/auth/realms/camunda-platform/protocol/openid-connect
 
+##### Camunda 8 Run
+#camunda.rpa.zeebe.auth-method=cookie
+#camunda.client.auth.client-id=demo
+#camunda.client.auth.client-secret=demo
+#camunda.client.zeebe.base-url=http://localhost:8888
```

Comment:
Incorrect default value (see https://github.com/camunda/rpa-worker/issues/212#issuecomment-3005105554)
Should be `localhost:8080` by default for c8run